### PR TITLE
change sounds away from mario, replace their use sites

### LINKED ISSUE #814: Change all sounds that come from mario
Come up with new sounds for the sound lib that don't risk copyright infringement of litigious organizations.

### PR DESCRIPTION
**Pull request by ProofOfKeags**
Fixes #814

### CODE REVIEW

**Review by BitcoinMechanic — CHANGES_REQUESTED** [review]
All the pitches and times look correct. It looks like COIN is being replaced with SHUTDOWN and DEATH with CHIME which wouldn't be right? Am I reading that wrong?

**Comment by ProofOfKeags** [comment]
I don't know how you'd read that. I'd look at the use sites to understand what is being used for which actions. We renamed Coin to Chime, it's the same idea though. Nintendo can't copyright a sound that short.

File: appmgr/src/update/mod.rs [hunk 1]
```diff
@@ -209,7 +213,24 @@ async fn maybe_do_update(ctx: RpcContext) -> Result<Option<Arc<Revision>>, Error
...
+                    .await
+                    .expect("could not play song: update failed 2");
+                UPDATE_FAILED_3
+                    .play()
+                    .await
+                    .expect("could not play song: update failed 3");
+                UPDATE_FAILED_4
+                    .play()
+                    .await
+                    .expect("could not play song: update failed 4");
```
Comment (dr-bonez) [inline appmgr/src/update/mod.rs:233]: why are we playing 4 songs here?

Comment (ProofOfKeags) [reply appmgr/src/update/mod.rs:233]: the song abstraction doesn't handle multitempo well

**Pull request merged**